crates/circuit/src/packed_instruction.rs

```diff
@@ -111,7 +111,7 @@ unsafe impl ::bytemuck::NoUninit for PackedOperationType {}
 /// * The pointed-to data must match the type of the discriminant used to store it.
 /// * `PackedOperation` must take care to forward implementations of `Clone` and `Drop` to the
 ///   contained pointer.
-#[derive(Debug)]
+#[derive(Debug, IntoPyObject, IntoPyObjectRef)]
 #[repr(transparent)]
 pub struct PackedOperation(usize);
```

Comment:
This really feels wrong - `PackedOperation` definitely should not ever be exposed to Python.  Aside from the data being completely meaningless to Python, implementing `IntoPyObjectRef` for it allows the holder to `Clone` the inner `usize`, which might represent ownership of a `Box` pointer.

Reply:
Removed in: https://github.com/Qiskit/qiskit/pull/13577/commits/132de0ad23929c2f8ff3c1296872eec5a1b075be